gatsby-config.js

```diff
@@ -58,7 +65,6 @@ const config = {
         mergeCachingHeaders: true,
       },
     },
-    'gatsby-transformer-remark',
```

Comment:
We had this transformer declared twice for some reason...

Reply:
It's like repeatedly pressing the elevator button.